File: testdata/jennies/rawtypes/enums/JavaRawTypes/src/main/java/com/grafana/foundation/enums/DashboardCursorSync.java
package com.grafana.foundation.enums;


// 0 for no shared crosshair or tooltip (default).
// 1 for shared crosshair.
// 2 for shared crosshair AND shared tooltip.
public enum DashboardCursorSync {
    OFF(0),
    CROSSHAIR(1),
    TOOLTIP(2);

    private Integer value;

    private DashboardCursorSync(Integer value) {
        this.value = value;
    }

    public Integer getValue() {
        return value;
    }
}

File: testdata/jennies/rawtypes/enums/CSharpRawTypes/src/Grafana/Foundation/Enums/DashboardCursorSync.cs
namespace Grafana.Foundation.Enums;

// 0 for no shared crosshair or tooltip (default).
// 1 for shared crosshair.
// 2 for shared crosshair AND shared tooltip.
public enum DashboardCursorSync
{
    Off = 0,
    Crosshair = 1,
    Tooltip = 2
}
